File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.js
const fs = require("fs");
const filePath =
  process.platform === "linux" ? "/dev/stdin" : "../../input.txt";
const inputs = fs.readFileSync(filePath, "utf-8").trim().split("\n");

const N = Number(inputs[0]);
const sizes = inputs[1].split(" ").map(Number);
const [T, P] = inputs[2].split(" ").map(Number);

// console.log(N);
// console.log(sizes);
// console.log(T, P);

/**
 * 1. 티셔츠 개수 : 각 value 돌며 Math.ceil(value / T)
 * 2. 펜 묶음 개수 : Math.floor(N / P)
 * 3. 한 자루씩 개수 : N - 2.
 */

// 1
let tCount = 0;
for (const size of sizes) {
  tCount += Math.ceil(size / T);
}
console.log(tCount);

// 2 + 3
const pen = Math.floor(N / P);
const individual = N - pen * P;
console.log(pen, individual);

File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.py

import math

N = int(input())
shirts = list(map(int,input().split(" ")))
T, P = map(int,input().split(" "))

# 티셔츠를 T장씩 최소 몇 묶음 주문
cnt = 0
for shirt in shirts:
  cnt += math.ceil(shirt / T)
  
print(cnt)
# 다음 줄에 펜을 P자루씩 최대 몇 묶음 주문할 수 있는지와, 그 때 펜을 한 자루씩 몇 개 주문하는지 구하세요.
print(N // P, N % P)
